include/swift/AST/Types.h

```diff
@@ -2985,16 +2986,11 @@ class AnyFunctionType : public TypeBase {
     // SWIFT_ENABLE_TENSORFLOW
     LLVM_NODISCARD
     ExtInfo withDifferentiabilityKind(
-        DifferentiabilityKind differentiability = DifferentiabilityKind::Normal)
+        DifferentiabilityKind differentiability)
     const {
-      switch (differentiability) {
-        case DifferentiabilityKind::NonDifferentiable:
-          return ExtInfo(Bits & ~DifferentiabilityMask);
-        case DifferentiabilityKind::Normal:
-          return ExtInfo(Bits & ~(0b10 << DifferentiabilityMaskOffset));
-        case DifferentiabilityKind::Linear:
-          return ExtInfo(Bits | DifferentiabilityMask);
-      }
+      unsigned clearedBits = Bits & ~(0b11 << DifferentiabilityMaskOffset);
```

Comment:
`(0b11 << DifferentiabilityMaskOffset)` is just equivalent to `DifferentiabilityMask`, so we should use that instead. With that, this function can be implemented with one return statement.